src/e2e/java/teammates/e2e/util/LNPResultsStatistics.java

```diff
@@ -0,0 +1,74 @@
+package teammates.e2e.util;
+
+import org.testng.Reporter;
+
+/**
+ * Represents the L&P test results statistics.
+ */
+public class LNPResultsStatistics {
+
+    private double errPct;
+    private double meanResTime;
+    private double pct1ResTime; // 90th percentile
+    private double throughput;
+    private int errorCount;
+    private int sampleCount;
+    private String resultsErrorMessage;
+    private String testName;
+
+    /**
+     * Display the feedback message based on LNP results statistics with the provided threshold.
+     */
+    public void displayLnpResultsStatistics(double errorRateLimit, double meanResTimeLimit) {
```

Comment:
This doesn't match the method content. How about something like `verifyLnpTestSuccess()`?
update javadocs too